src/fides/api/api/v1/endpoints/system.py

```diff
@@ -389,37 +389,62 @@ async def ls(  # pylint: disable=invalid-name
     data_uses: Optional[List[FidesKey]] = Query(None),
     data_categories: Optional[List[FidesKey]] = Query(None),
     data_subjects: Optional[List[FidesKey]] = Query(None),
+    dnd_relevant: Optional[bool] = Query(None),
+    show_hidden: Optional[bool] = Query(False),
 ) -> List:
     """Get a list of all of the Systems.
-    If any pagination parameters (size or page) are provided, then the response will be paginated
-    & provided filters (search, taxonomy fields) will be applied.
+    If any parameters or filters are provided the response will be paginated and/or filtered.
```

Comment:
Can we check with someone in the frontend team that this change in behavior won't affect any existing functionality? e.g if we're using this endpoint anywhere in the frontend without pagination params but some filters like `data_categories`, then this will break cause that response will now be paginated but the frontend won't expect that

Reply:
well yes but we can't release something if it's broken 😅 I agree that it _shouldn't_ be the case that the frontend was sending the filter params but not the pagination ones, but we should check with the frontend team just to make sure , cause returning a paginated response when the frontend isn't expecting would break it

Reply:
we only use the systems api in one place as far as I can tell https://github.com/ethyca/fides/blob/ea84dc29bfd1b5e51b41995a9ec4e52cc311aeab/clients/admin-ui/src/pages/systems/index.tsx#L105

https://github.com/search?q=repo%3Aethyca%2Ffides+%22getSystems%22&type=code

I'll ask FE if anyone knows of more places but I don't think we have a place that's expecting an un-paginated response with filters that it also doesn't actually want